datafusion/core/src/physical_plan/file_format/parquet.rs

```diff
@@ -515,10 +516,10 @@ fn read_partition(
                     let err_msg =
                         format!("Error reading batch from {}: {}", partitioned_file, e);
                     // send error to operator
-                    send_result(
+                    let _ = send_result(
                         &response_tx,
                         Err(ArrowError::ParquetError(err_msg.clone())),
-                    )?;
```

Comment:
Here the send_result error should not overwrite the original ArrowError, so I ignored it.